portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,24 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Lina's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <ul>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="misc.html">For Fun</a></li>
+    </ul>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 id="title">Lina's Portfolio</h1>
+      <p>Welcome to my portfolio! My name is Lina, and I'm a Latina in Tech currently interning at Google as a STEP intern. I am a rising Junior at Yale!</p> 
```

Comment:
Nit: There's not technically a max line length in HTML, but consider adding a line break at 80 or 100 to make this more readable.